Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -66,6 +86,34 @@ protected override void OnStart()
             _wallRunBehaviour?.Initialize(this, base.GetMovementInput, ResetJumpCount);
         }
 
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+
+        protected override Vector2 GetMouseLookInput()
```

Comment:
```suggestion
                protected override Vector2 GetMouseLookInput()
        {
            if (IsDebugWindowOpen)
            {
                return Vector2.zero;
            }
            
            return base.GetMouseLookInput();
        }

        protected override void OnCursorLock(InputAction.CallbackContext context)
        {
            if (IsDebugWindowOpen)
            {
                return;
            }
            base.OnCursorLock(context);
        }
```

Reply:
Oh I did not know you could do suggested changes O_o